includes/class-amp-theme-support.php

```diff
@@ -2463,6 +2470,59 @@ public static function enqueue_assets() {
 		wp_styles()->add_data( 'amp-default', 'rtl', 'replace' );
 	}
 
+	/**
+	 * Enqueue AMP paired browsing client script.
+	 *
+	 * @return void
+	 */
+	public static function enqueue_paired_browsing_client() {
+		wp_enqueue_script(
+			'amp-paired-browsing-client',
+			amp_get_asset_url( '/js/amp-paired-browsing-client.js' ),
+			[ 'admin-bar' ],
+			AMP__VERSION,
+			true
+		);
+	}
+
+	/**
+	 * Includes a custom template that acts as an interface to facilitate a side-by-side comparison of a
+	 * non-AMP page and its AMP version to review any discrepancies.
+	 *
+	 * @param string $template Path of the template to include.
+	 * @return string Custom template if in paired browsing mode, else the supplied template.
+	 */
+	public static function serve_paired_browsing_experience( $template ) {
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		if ( ! isset( $_GET[ self::PAIRED_BROWSING_QUERY_VAR ] ) ) {
+			return $template;
+		}
+
+		if ( ! is_admin_bar_showing() ) {
+			wp_die( esc_html__( 'The admin bar must be showing to use paired browsing mode.', 'amp' ) );
+		}
+
+		wp_enqueue_style(
+			'amp-paired-browsing-app',
+			amp_get_asset_url( '/css/amp-paired-browsing-app-compiled.css' ),
+			[ 'dashicons' ],
+			AMP__VERSION
+		);
+
+		wp_enqueue_script(
```

Comment:
Same here with dependencies list & version

Reply:
Thanks for the review, @swissspidy!